ui-components/templates/products.html

```diff
@@ -0,0 +1,16 @@
+<div class="products-title">All poducts</div>
+<div class="products">
+  <div class="product-card" ng-repeat="product in products">
+    <div class="image"><img ng-src="{{product.image}}" alt=""></div>
+    <div class="title">
+      <a ui-sref="main.details({id: product.id})">{{product.name}}</a>
+    </div>
+    <div class="price">
+      Price: {{product.price}}
+    </div>
+    <div class="add-to-basket">
+      <button ng-if="!product.inOrder" class="btn btn-primary" ng-click="addToCart(product)"><span class="glyphicon glyphicon-shopping-cart"></span>Add to cart</button>
+      <div ng-if="product.inOrder">This product is already in the cart</div>
+    </div>
+  </div>
+</div>
```

Comment:
You might want to add IDE option like `add new line at the end` to make github happy